models/topics.js

```diff
@@ -0,0 +1,12 @@
+const db = require('../db/connection');
+
+exports.selectTopics = () => {
+
+    let sql = `select *  from topics ; `;
+    
```

Comment:
Although the lowercase SQL statement won't stop this running, please update this to uppercase as per convention.